src/main/kotlin/com/hubelias/parkingmeter/port/adapter/rest/ParkedVehiclesEndpoint.kt

```diff
@@ -0,0 +1,49 @@
+package com.hubelias.parkingmeter.port.adapter.rest
+
+import com.hubelias.parkingmeter.application.ParkingMeterFacade
+import org.springframework.hateoas.mvc.ControllerLinkBuilder.linkTo
+import org.springframework.http.HttpHeaders
+import org.springframework.http.HttpStatus
+import org.springframework.http.ResponseEntity
+import org.springframework.stereotype.Controller
+import org.springframework.web.bind.annotation.*
```

Comment:
takie importy utrudniają review bo nie zawsze widać co skąd jest

Reply:
Być może :) Przyznam, że zawsze byłem zadowolony w jaki sposób IDEA porządkuje importy przy domyślnych ustawieniach, że wraz z reformatem robi się to u mnie automatycznie przy każdym commicie. Ale rzeczywiście może to czasem utrudniać review :)